docs/contributing.rst

```diff
@@ -312,6 +312,33 @@ Please:
 .. _PEP 8 - Style Guide for Python Code:
   https://www.python.org/dev/peps/pep-0008
 
+Mypy type annotations
+=====================
+
+Certbot uses the ``mypy`` static type checker. Python 3 natively supports official type annotations, which can then be tested for consistency using ``mypy``. Python 3 doesn’t, but type annotations can be `added in comments`_. ``mypy`` does some type checks even without type annotations; we can find bugs in Certbot even without a fully annotated codebase.
```

Comment:
nit: I think it'd be cool if at least the first usage of `mypy` here linked to the mypy documentation, but this is an improvement we can always add later.

Reply:
s/Python 3 doesn't/Python 2 doesn't

(a find replace command like that probably needs some backslashes for the apostrophes, but I think your ability to parse that is probably better than a command like `sed`.